experiments/cbow.json

```diff
@@ -0,0 +1,79 @@
+{
+  "data_file": "HOME/work/experiments/nlp/data/books/frankenstein_with_splits.csv",
+  "model_state_file": "modelCBOW.pth",
```

Comment:
are all these string properties used? maybe we should clean out the unused ones...after merging i could add a warning message for unused config params. that would make it easier. we could postpone cleanup til then.

Reply:
we don't use model checkpointing in the `BasicTrainer` so `model_state_file` can be omitted for now